lib/api/clusters.ts

```diff
@@ -155,8 +155,9 @@ export const getActiveClusterCountByZkvmId = async () => {
   return result
 }
 
-// Those machines that at least have 1 proof submitted with status "proved"
 export const getActiveMachineCount = async () => {
+  const sevenDaysAgo = sql`NOW() - INTERVAL '7 days'`
```

Comment:
ooo this `sql` function looks helpful... 

@pettinarip Any reason we couldn't extract this to the top since the exact sql pattern is used three times in this file? Won't let that block, but just an idea